nwb_web_gui/dashapps/pages/converter/converter_utils/forms.py

```diff
@@ -301,51 +300,44 @@ def get_definitions_items(self, ref_key, extra_key, sublist=False):
 
         return form
 
-    def get_string_field_input(self, value, input_id):
+    @staticmethod
+    def get_string_field_input(value, input_id):
 
-        if 'description' in value:
-            description = value['description']
-        else:
-            description = ''
+        description = getattr(value, 'description', '')
 
         if 'enum' in value:
             input_values = [{'label': e, 'value': e} for e in value['enum']]
-            if 'default' in value:
-                default = value['default']
-            else:
-                default = ''
-            form_input = dcc.Dropdown(
+            default = getattr(value, 'default', '')
+            return dcc.Dropdown(
```

Comment:
`return` as early as you can within functions. This makes the function less indented and therefore less complex.